snuba/cli/consumer.py

```diff
@@ -133,6 +134,8 @@ def consumer(
     setup_logging(log_level)
     setup_sentry()
     logger.info("Consumer Starting")
+
+    storage_name = storage_names[0]
```

Comment:
If you are (temporarily) just picking the first from the list and discarding the rest, please raise an exception if any more than 1 is passed. It may be very unexpected behavior.